server.js

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-const { FUNCTION_URI, HOST, RIFF_FUNCTION_INVOKER_PROTOCOL, HTTP_PORT } = process.env;
+const { FUNCTION_URI, RIFF_FUNCTION_INVOKER_PROTOCOL } = process.env;
+const PORT = process.env.PORT || 8080;
```

Comment:
If we preserve support for HTTP_PORT (if set) we can avoid this being a breaking change. Also made the value a string since all env vars are implicitly strings.

```suggestion
const PORT = process.env.HTTP_PORT || process.env.PORT || '8080';
```

Reply:
@scothis good point!